src/main/java/uk/nhs/hee/tis/trainee/forms/interceptor/AdminIdentityInterceptor.java

```diff
@@ -60,6 +61,8 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
         adminIdentity.setEmail(email);
         Set<String> adminGroups = AuthTokenUtil.getAttributes(authToken, GROUPS_ATTRIBUTE);
         adminIdentity.setGroups(adminGroups);
+        Set<String> adminRoles = AuthTokenUtil.getAttributes(authToken, ROLES_ATTRIBUTE);
+        adminIdentity.setRoles(adminRoles);
```

Comment:
Added to root AdminIdentity class, though it could be limited to just the AdminLtftIdentity class, since its only used there. It felt like it made sense to include it here since it fitted with the groups and involved less code duplication in the subclass.